ci/unit.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.13
```

Comment:
Should we be going to version 1.14? Same with the go.mod file

There is even go version 1.15.* now too, so arguably bump to latest version?

Reply:
Well, my boss i on Holiday, so i'm going with the version stated in the setting up your Mac guide.
It will be easy enough to adjust later, if thats what she wants.